CONTRIBUTING.md

```diff
@@ -164,6 +186,46 @@ After you get your container up and running, you can test it by adding test asse
 
 Finally, commit your changes and submit a PR - we'll take a look at it, provide any needed feedback, and then merge it in. We appreciate any and all feedback!
 
+### Adding a Database Definition to an existing Container
+
+VS Code Dev Containers allow fairly straightforward modification for allowing multiple containers to run together. This is useful when adding database support to an existing container.
+
+1. Create a new copy of an existing language definition folder, and hyphenate the database name.
+   1. For example, `java` becomes `java-postgres`.
+2. Create a new `docker-compose.yml` file.
+   1. Add the existing `Dockerfile` to the `docker-compose` as an `app`. This will be the Dev Container.
+   2. Add the definition for database to the `docker-compose`, version tagging it or using the `latest`. We don't use custom images for databases, as this is generally not needed.
+   3. It will likely be necessary to network the containers so that the `app` can connect to the `db`. This can normally be done by adding the following to the `app` definition:
+
+    ```yaml
+        network_mode: service:db
+    ```
+
+3. Modify the `devcontainer.json` with any launch options or VS Code specific `settings.json` options that may be needed. 
+   1. For example, when working with a SQL database, it might be necessary to modify the settings with something like the following:
+   ```json
+   		"sqltools.connections": [
+			{
+			  "name": "Container database",
+			  "driver": "PostgreSQL",
+			  "previewLimit": 50,
+			  "server": "localhost",
+			  "port": 5432,
+			  // NOTE: database/username/password should match values in docker.compose.yml
+			  "database": "postgres",
+			  "username": "postgres",
+			  "password": "postgres"
+			}
+		  ],
+    ```
+    2. It might be necessary to add additional VS Code Extensions to allow for an easier dev experience.
+4. Modify the existing Tests or existing Application to ensure that the following criteria are met:
```

Comment:
I'll try to force the suggestion to be a test rather than the app. Since CI would cover test if test.sh is present.